back/identify/LinkedIn/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+# Create your models here.
+
+
+class LinkedIn(models.Model):
```

Comment:
It's good to put blank true on some fields like summary or country or city or industry name  ...
They might be empty("") and not null